kinetic/distribution.yaml

```diff
@@ -7718,6 +7718,22 @@ repositories:
       url: https://github.com/larics/smartek_camera.git
       version: master
     status: maintained
+  soem:
+    doc:
+      type: git
+      url: https://github.com/smits/soem.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/smits/soem-gbp.git
+      version: 1.3.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Just checking: did you follow the instructions to provide the necessary access for for the @ros-pull-request-builder user?